scripts/tests.e2e.sh

```diff
@@ -3,116 +3,47 @@ set -e
-      echo "Usage: ${0} [VERSION_1] [VERSION_2]" >> /dev/stderr
-      exit 255
-    fi
+CAMINO_NODE_PATH="${1-}"
+if [[ -z "${CAMINO_NODE_PATH}" ]]; then
+  echo "Missing CAMINO_NODE_PATH argument!"
+  echo "Usage: ${0} [CAMINO_NODE_PATH]" >> /dev/stderr
+  exit 255
 fi
 
-echo "Running e2e tests with:"
-echo VERSION_1: ${VERSION_1}
-echo VERSION_2: ${VERSION_2}
+echo "Running e2e tests:"
 
 TEMP_PATH=/tmp
 
-if [ ! -f ${TEMP_PATH}/camino-node-v${VERSION_1}/camino-node ]
-then
-    ############################
-    # download camino-node
-    # https://github.com/chain4travel/camino-node/releases
-    GOARCH=$(go env GOARCH)
-    GOOS=$(go env GOOS)
-    DOWNLOAD_URL=https://github.com/chain4travel/camino-node/releases/download/v${VERSION_1}/camino-node-linux-${GOARCH}-v${VERSION_1}.tar.gz
-    DOWNLOAD_PATH=${TEMP_PATH}/camino-node.tar.gz
-    if [[ ${GOOS} == "darwin" ]]; then
-      DOWNLOAD_URL=https://github.com/chain4travel/camino-node/releases/download/v${VERSION_1}/camino-node-macos-v${VERSION_1}.zip
-      DOWNLOAD_PATH=${TEMP_PATH}/camino-node.zip
-    fi
-
-    rm -rf ${TEMP_PATH}/camino-node-v${VERSION_1}
-    rm -rf ${TEMP_PATH}/camino-node-build
-    rm -f ${DOWNLOAD_PATH}
-
-    echo "downloading camino-node ${VERSION_1} at ${DOWNLOAD_URL}"
-    curl -L ${DOWNLOAD_URL} -o ${DOWNLOAD_PATH}
-
-    echo "extracting downloaded camino-node"
-    if [[ ${GOOS} == "linux" ]]; then
-      tar xzvf ${DOWNLOAD_PATH} -C ${TEMP_PATH}
-    elif [[ ${GOOS} == "darwin" ]]; then
-      unzip ${DOWNLOAD_PATH} -d ${TEMP_PATH}/camino-node-build
-      mv ${TEMP_PATH}/camino-node-build/build ${TEMP_PATH}/camino-node-v${VERSION_1}
-    fi
-fi
-
-if [ ! -f ${TEMP_PATH}/camino-node-v${VERSION_2}/camino-node ]
-then
-    ############################
-    # download camino-node
-    # https://github.com/chain4travel/camino-node/releases
-    GOARCH=$(go env GOARCH)
-    GOOS=$(go env GOOS)
-    DOWNLOAD_URL=https://github.com/chain4travel/camino-node/releases/download/v${VERSION_2}/camino-node-linux-${GOARCH}-v${VERSION_2}.tar.gz
-    DOWNLOAD_PATH=${TEMP_PATH}/camino-node.tar.gz
-    if [[ ${GOOS} == "darwin" ]]; then
-      DOWNLOAD_URL=https://github.com/chain4travel/camino-node/releases/download/v${VERSION_2}/camino-node-macos-v${VERSION_2}.zip
-      DOWNLOAD_PATH=${TEMP_PATH}/camino-node.zip
-    fi
-
-    rm -rf ${TEMP_PATH}/camino-node-v${VERSION_2}
-    rm -rf ${TEMP_PATH}/camino-node-build
-    rm -f ${DOWNLOAD_PATH}
-
-    echo "downloading camino-node ${VERSION_2} at ${DOWNLOAD_URL}"
-    curl -L ${DOWNLOAD_URL} -o ${DOWNLOAD_PATH}
-
-    echo "extracting downloaded camino-node"
-    if [[ ${GOOS} == "linux" ]]; then
-      tar xzvf ${DOWNLOAD_PATH} -C ${TEMP_PATH}
-    elif [[ ${GOOS} == "darwin" ]]; then
-      unzip ${DOWNLOAD_PATH} -d ${TEMP_PATH}/camino-node-build
-      mv ${TEMP_PATH}/camino-node-build/build ${TEMP_PATH}/camino-node-v${VERSION_2}
-    fi
-fi
+mkdir $TEMP_PATH/camino-node-1 -p
+mkdir $TEMP_PATH/camino-node-2 -p
```

Comment:
do we still need 2 paths? till now there were in theory 2 node versions tested by the e2e tests. But as I understand now there's only 1 node version being tested.

Reply:
Yep, only one version for now. Initially I wanted to keep 2 paths so it will remind that we can use to different versions, but e2e.test executable will have 2 path args anyway, so reminder remains. I'll remove second mkdir and cp